python/lsst/ts/aos/utils/m1m3/m1m3_aav.py

```diff
@@ -310,8 +320,7 @@ async def find_applicable_times(self) -> None:
                     ret.append([start, end])
                 else:
                     logging.warning(
-                        f"Interval {start.isot} - {end.isot} "
-                        "mirror was not raised, ignoring."
+                        f"Interval {start} - {end} " "mirror was not raised, ignoring."
```

Comment:
This comment is a little bit unclear on my opinion. Mayb something like "Ignoring, sinc the mirror was not raised during the interval start - end"

Reply:
What about that?
```
-                        f"Interval {start} - {end} " "mirror was not raised, ignoring."
+                        f"Ignoring interval {start} - {end} since the mirror was not raised."
```